doc/website/getting-started/installation.md

```diff
@@ -7,9 +7,9 @@ All iceoryx libraries are deployed as independent CMake packages. Posh is using
 ### :octicons-package-dependencies-16: Dependencies
 
 - 64-bit hardware (e.g. x86_64 or aarch64; 32-bit hardware might work, but is not supported)
-- [CMake](https://cmake.org), 3.10 or later
+- [CMake](https://cmake.org), 3.16 or later
 - One of the following compilers:
-    - [GCC](https://gcc.gnu.org), 7.4 or later (5.4 currently supported too)
+    - [GCC](https://gcc.gnu.org), 8.3 or later (5.4 currently supported too)
```

Comment:
Shouldn't we state that we support gcc >= 5.4? Now it sounds like, 8.3 or greater is supported and by the way also 5.4 but what is with 6 and 7?

Reply:
I understand your argumentation but then I would rephrase it like: 5.4 or later (verified via CI 8.3, ...)
The reason is that there are a lot of versions which we are not testing, see:
8.4, 8.5, 9.0, 9.1, 9.2, 9.3, 9.4, 10.0, 10.1 .... you get the idea, only a minor subset.

And what we do in the CI is completely reasonable, we test the oldest compiler which we are supporting, something in the middle and the newest one.

Reply:
Well, this is an installation manual addressing users that are new to iceoryx. Giving them now the range "5.4 or later" will somehow imply that we are fully supporting versions like 6 or 7 too.  Version 5.4 is only an exception for QNX SDP 7.0 that we made. Version 8 is guaranteed to work by CI.

Additionally I would avoid to maintain another list of version numbers of compilers that are tested in the CI because they are subject of change while testing the latest compiler.